book.bib

```diff
@@ -1148,6 +1147,16 @@ @inproceedings{adams2016a
   pages = {78--90},
   volume = {5},
 }
+@article{laukkanen2017a,
```

Comment:
How did you find this paper? This information is needed for the Appendix, because now this paper will be cited without being included after following our protocol :sweat_smile:

Reply:
Found it through the references of another paper by the same author I believe. We can just place it in the list with included studies right?

Reply:
If you found it via another paper, can you please add in table 1 how exactly you found it, just to be complete? Besides this, it should indeed also be added to the list of included studies. :slightly_smiling_face:

Reply:
Done